compiler/src/dotty/tools/dotc/Driver.scala

```diff
@@ -64,19 +64,31 @@ class Driver {
 
   protected def sourcesRequired: Boolean = true
 
-  def setup(args: Array[String], rootCtx: Context): (List[AbstractFile], Context) = {
+  protected def command: CompilerCommand = ScalacCommand
+
+  /** Setup context with initialized settings from CLI arguments, then check if there are any settings that
+   *  would change the default behaviour of the compiler.
+   *
+   *  @return An Option of tuple of List and Context.
```

Comment:
That line isn't really necessary since it just repeats the type without adding information.